package.json

```diff
@@ -4,14 +4,14 @@
   "scripts": {
     "postinstall": "npm run bootstrap",
     "prebootstrap": "npm run ensure-env",
-    "bootstrap": "lerna bootstrap && lerna run build --scope @freecodecamp/curriculum",
```

Comment:
Does`seed: challenges` use builded curriculum?

Reply:
Curriculum package should be deleted from   seed `package.json`

Reply:
Removed from `seed` and `api-server`